app/src/main/java/com/yeonae/chamelezone/view/place/adapter/PlaceReviewTabRvAdapter.kt

```diff
@@ -29,28 +46,26 @@ class PlaceReviewTabRvAdapter(private val reviewList: ArrayList<Review>) :
     override fun getItemCount(): Int =
         reviewList.size
 
-    override fun onBindViewHolder(holder: PlaceReviewViewHolder, position: Int) =
-        holder.bind(reviewList[position])
+    override fun onBindViewHolder(holder: PlaceReviewViewHolder, position: Int) {
+        if (::moreButtonListener.isInitialized) {
+            holder.bind(reviewList[position], moreButtonListener)
+        }
+    }
 
     fun addData(addDataList: List<Review>) {
         reviewList.clear()
         reviewList.addAll(addDataList)
         notifyDataSetChanged()
     }
 
-    fun setItemClickListener(clickListener: OnItemClickListener){
-        itemClickListener = clickListener
-    }
-
-
-
     inner class PlaceReviewViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
         private val userId: TextView = itemView.findViewById(R.id.tv_user_id)
         private val reviewDate: TextView = itemView.findViewById(R.id.tv_review_date)
         private val reviewImg: ImageView = itemView.findViewById(R.id.iv_review_img)
         private val reviewContent: TextView = itemView.findViewById(R.id.tv_review_content)
+        private val moreReviewImg: LinearLayout = itemView.findViewById(R.id.iv_image)
 
-        fun bind(review: Review) {
+        fun bind(review: Review, moreButtonListener: MoreButtonListener) {
```

Comment:
```suggestion
        fun bind(review: Review, moreButtonListener: MoreButtonListener) {
```
inner class로 사용하는데 'MoreButtonListener' 만 굳이 인자로 받을 필요 있을까요?

Reply:
수정했습니다
04a8db2